docs/designs/013-remediation-plugins.md

```
@@ -0,0 +1,258 @@
+```
+
+**Plugin Side**:
+- Handle cloud provider SDK calls and any custom logic (approvals, logging, etc.)
+- Deploy as separate Kubernetes Deployment/Service
+- Example AWS server implementing SendRebootSignal:
+
+```go
+package main
+
+import (
+	"context"
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)